filesystem/config.pan

```diff
@@ -50,6 +49,20 @@ function filesystem_layout_mod = {
   SELF;
 };
 
+# Helper function for calculating partiton names. Takes an escaped disk name
+# and partition index, and returns the escaped partition name
+function disk_part_name = {
+    disk = ARGV[0];
+    part_num = ARGV[1];
+
+    if (exists("/hardware/harddisks/" + disk + "/part_prefix")) {
+        part_prefix = value("/hardware/harddisks/" + disk + "/part_prefix");
+    } else {
+        part_prefix = "";
```

Comment:
the default should be 'p' when the diskname ends  with a digit, eg for nvme disks

Reply:
I disagree. It would be sub-optimal to add such custom logic everywhere. `part_prefix` was invented exactly for this reason - if you need the 'p', then set `part_prefix` (we do :smile:)